source/default/_data/scssVariables.json

```diff
@@ -36,6 +36,7 @@
       "Roboto",
       "Helvetica Neue",
       "Arial",
+      "Noto Sans",
```

Comment:
Just to be super clear, this is coming from the Bootstrap 4 update.

Reply:
Indeed.